src/index.html

```diff
@@ -17,61 +17,430 @@
   </head>
   <body>
     <header class="header">
-      <a class="header__logo" href="">
-        <img src="./images/logo.png" alt="Moyo logo">
+      <a href="#home" class="logo">
+        <img src="./images/logo.png" alt="logo">
       </a>
 
       <nav class="nav">
         <ul class="nav__list">
-          <li class="nav__item nav__item--is-active">
-            <a href="" class="nav__link">Apple</a>
+          <li class="nav__item">
+            <a href="#apple" class="nav__link is-active">Apple</a>
           </li>
+
           <li class="nav__item">
-            <a href="" class="nav__link">Samsung</a>
+            <a href="#samsung" class="nav__link">Samsung</a>
           </li>
+
           <li class="nav__item">
-            <a href="" class="nav__link">Smartphones</a>
+            <a href="#smartphones" class="nav__link">Smartphones</a>
           </li>
+
           <li class="nav__item">
-            <a href="" class="nav__link">Laptops & computers</a>
+            <a
+              href="#laptops & komputers"
+              data-qa="nav-hover"
+              class="nav__link"
+            >
+              Laptops & Computers
+            </a>
           </li>
+
           <li class="nav__item">
-            <a href="" class="nav__link">Gadgets</a>
+            <a href="#gadgets" class="nav__link">Gadgets</a>
           </li>
+
           <li class="nav__item">
-            <a href="" class="nav__link">Tablets</a>
+            <a href="#tablets" class="nav__link">Tablets</a>
           </li>
+
           <li class="nav__item">
-            <a href="" class="nav__link">Photo</a>
+            <a href="#photo" class="nav__link">Photo</a>
           </li>
+
           <li class="nav__item">
-            <a href="" class="nav__link">Video</a>
+            <a href="#video" class="nav__link">Video</a>
           </li>
         </ul>
       </nav>
     </header>
-
-    <main class="catalog">
-      <div class="catalog__card">
-        Product card
-      </div>
-      <div class="catalog__card">
-        Product card
-      </div>
-      <div class="catalog__card">
-        Product card
-      </div>
-      <div class="catalog__card">
-        Product card
-      </div>
-      <div class="catalog__card">
-        Product card
-      </div>
-      <div class="catalog__card">
-        Product card
-      </div>
-      <div class="catalog__card">
-        Product card
+    <main class="main">
+      <div class="box">
+        <div class="card box__card" data-qa="card">
+          <img
+            class="card__img"
+            src="./images/imac.jpeg"
+            alt="APPLE A1419 iMac 27"
+          >
+  
```

Comment:
You didn't need to change the html file. Your page should look like this 
<img width="1680" alt="image" src="https://user-images.githubusercontent.com/99403198/222131888-a9ff0a9e-ac5a-40ee-aec5-8da416d90c4d.png">